packages/stacked_cli/lib/src/templates/app/lib/ui/setup/setup_bottom_sheet_ui.dart.stk

```diff
@@ -1,18 +1,23 @@
 import 'package:flutter/material.dart';
 import 'package:{{packageName}}/{{{relativeLocatorPath}}}';
 import 'package:{{packageName}}/enums/bottom_sheet_type.dart';
-import 'package:{{packageName}}/ui/bottom_sheets/notice_sheet/notice_sheet.dart';
+import 'package:{{packageName}}/ui/bottom_sheets/notice/notice_sheet.dart';
+// @stacked-bottom-sheet-import
 import 'package:stacked_services/stacked_services.dart';
 
+typedef BottomSheetBuilder = Widget Function(
```

Comment:
Nice update

Reply:
I was happy with the change and I known u would like it 😄